### ISSUE TITLE
Crash with With El Capitan (10.11.6)

### ISSUE THREAD

**Issue by HorstAlbiez** [BTCPrivate/electrum-btcp#69]
Application crashes right after startup.

**Comment by donshin** [comment]
@HorstAlbiez have you experienced this with the latest release (version 1.1.1)?

**Comment by HorstAlbiez** [comment]
Yes!
It works with Version P!1.0.0b_macos_10.11 

Regards
Horst

**Comment by donshin** [comment]
@HorstAlbiez I'm not familiar with that release- where did you download P!1.0.0b_macos_10.11? In any case, can you provide more info about your mac model and CPU details?

**Comment by HorstAlbiez** [comment]
It was mentioned (along with a download link) at Github as a hint for one of the first „issues“. One of the users ran into the same problem with 10.11.


Von meinem iPhone gesendet

**Comment by donshin** [comment]
@HorstAlbiez I do not believe that was an official release from us. In any case, can you provide more info about your mac model and CPU details?

**Comment by HorstAlbiez** [comment]
Here information about my MAC (it’s an old one but still running ;-))





The issue was discussed here:
I downloaded the (working) version from the link mentioned in the discussion.

**Comment by donshin** [comment]
@HorstAlbiez neither mac details nor link seemed to come through in your previous post. Can you check and edit or re-post please?

**Comment by HorstAlbiez** [comment]
Can’t you see the screenshots?
I included 2 screenshots...

Von meinem iPhone gesendet

**Comment by HorstAlbiez** [comment]
Again...

Von meinem iPhone gesendet

Anfang der weitergeleiteten Nachricht:

**Comment by donshin** [comment]
Unfortunately the screenshots are not posted in GitHub. Could you please type out the info?

**Comment by HorstAlbiez** [comment]
Ok

Von meinem iPhone gesendet

**Comment by HorstAlbiez** [comment]
My configuration:
  
  Modellname:	iMac
  Modell-Identifizierung:	iMac7,1
  Prozessortyp:	Intel Core 2 Duo
  Prozessorgeschwindigkeit:	2,4 GHz
  Anzahl der Prozessoren:	1
  Gesamtanzahl der Kerne:	2
  L2-Cache:	4 MB
  Speicher:	6 GB
  Busgeschwindigkeit:	800 MHz
  Boot-ROM-Version:	IM71.007A.B03
  SMC-Version (System):	1.21f4
  Seriennummer (System):	W873615AX8A
  Hardware-UUID:	00000000-0000-1000-8000-001B639DBFB2


  Systemversion:	OS X 10.11.6 (15G20015)
  Kernel-Version:	Darwin 15.6.0
  Startvolume:	Macintosh SSD
  Startmodus:	Normal
  Gerätename:	Horst Albiezs Computer
  Benutzername:	Horst Albiez (horstalbiez)
  Sicherer virtueller Speicher:	Aktiviert
  Systemintegritätsschutz:	Aktiviert
  Zeit seit dem Start:	7 Tage 10:55



Her the discussion in Github issue #19:

https://github.com/BTCPrivate/electrum-btcp/issues/19 <https://github.com/BTCPrivate/electrum-btcp/issues/19>

**Comment by HorstAlbiez** [comment]
Von meinem iPhone gesendet

Anfang der weitergeleiteten Nachricht:
